src/shared/components/iframe-runtime.tsx

```diff
@@ -117,9 +121,24 @@ export const IframeRuntime: React.FC<IProps> =
         phoneRef.current.disconnect();
       }
     };
-  },[addLocalLinkedDataListener, authoredState, logRequestData, report, setHint, url, initMessage]);
+  },[addLocalLinkedDataListener, authoredState, logRequestData, report, setHint, url, initMessage, view]);
 
-  const iframeStyle = iframeStyling ?? {width: "100%", height: iframeHeight, border: "none"};
+  let scaledIframeStyle = undefined;
+  if (scale && report && view !== "standalone") {
```

Comment:
Rather than pass the `view` in here. It would be better for the component using this IframeRuntime to not set the scale when the view is standalone. That way this IframeRuntime component doesn't need to know about this view property.

Reply:
@scytacki Do the latest changes make sense? I had thought of doing that before, but lines 89 and 103 in runtime.tsx seemed a little on the too cognitively complex side. Maybe not, though? Or maybe there's a better way to do it that I'm not thinking of?

Reply:
I think the approach in runtime.tsx is OK.
You could pull it out to a `leftScale` and `rightScale` variable so the logic is not inside of the jsx. For example:
```
let leftScale = undefined;
let rightScale = undefined;
if (report && view !== "standalone") {
  leftScale = leftReportScale;
  rightScale = rightReportScale;
}
```
But I think what you've got might be easier to read than what I put above.